ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1262,6 +1261,9 @@ def modify_tls(self, restart_service):
         )
         self.master.run_command(cmd)
 
+    @pytest.mark.skipif((osinfo.id == 'rhel' and osinfo.version_number > (9,0)),
```

Comment:
According to the comment the annotation needs to apply from RHEL 9.0, the condition should be `>=` instead of `>`.